dependency.lic

```diff
@@ -1220,28 +1242,65 @@ def stop_autostart(script)
   end
 end
 
+def set_dev_mode
+  echo "Now turning dev mode on"
+  echo "Developer mode now set to: #{$manager.developer = true}"
```

Comment:
I'm not a fan of doing the assignment in the string interpolation, but it won't hold up merging.